tuist/tuist
# Introduce mergeable libraries

Resolves #5296

### Short description 📝

With the introduction of mergeable libraries, we no longer need to embed dynamic libraries as part of the final app target when it is declared as mergeable and the libraries can be merged. The code in this PR is going to detect the changes and avoid embedding them into the final target.

### How to test the changes locally 🧐

1. Create a target (A) which has been compiled with -make_mergeable linker flag (available on Xcode 15+)
2. Set A as dependency on the application and enable MERGED_BINARY_TYPE to manual

The expected setup should be having A as dependency but not embedded into the app

### Contributor checklist ✅


### Reviewer checklist ✅
